backend/tracim_backend/lib/webdav/design.py

```diff
@@ -124,6 +126,16 @@
     'copy' : 'Item copied',
 }
 
+THREAD_MESSAGE = """
+  <p>
+    {posting_time},{comment_owner} wrote :
```

Comment:
by the way, there are no space before a colon: `Michel wrote :` => `Michel wrote:` (space removed before `:`)

Reply:
Done